src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -72,6 +78,24 @@ export class RundownComponent implements OnInit, OnDestroy, OnChanges {
       .catch(error => this.logger.data(error).error('Failed subscribing to rundown actions changes.'))
   }
 
+  public scrollViewToSegment(event: PartEvent): void {
+    const element: HTMLElement | null = document.getElementById(event.segmentId)
+    if (!element || this.isInsideViewport(element)) {
+      return
+    }
+
+    document.getElementById(event.segmentId)?.scrollIntoView({
```

Comment:
You already fetch the `element` on line 82. No need to fetch it again

Reply:
Good point! Changed!